tests/platform_tests/daemon/test_pcied.py

```diff
@@ -92,12 +92,15 @@ def collect_data(duthost):
     dev_summary_status = duthost.get_pmon_daemon_db_value(pcie_devices_status_tbl_key, status_field)
     return {'status': dev_summary_status, 'devices': dev_data}
     
-def wait_data(duthost):
+def wait_data(duthost, expected_key_count):
     class shared_scope:
         data_after_restart = {}
     def _collect_data():
         shared_scope.data_after_restart = collect_data(duthost)
-        return bool(shared_scope.data_after_restart['devices'])
+        device_keys_found = len(shared_scope.data_after_restart['devices'])
+        if device_keys_found != 0:
+            logger.info("Expected PCIE device keys :{}, Current device key count {}".format(db_key_count,device_keys_found))
```

Comment:
where is the definition of db_key_count?

Reply:
Fixed the issue. It should be expected_key_count